packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts

```diff
@@ -195,6 +195,15 @@ describe("FtpMvsApi", () => {
         expect(DataSetUtils.deleteDataSet).toBeCalledTimes(1);
         expect(MvsApi.releaseConnection).toBeCalled();
     });
+    it("should give error copy datasets.", async () => {
+        try {
+            await MvsApi.copyDataSet(null, null);
+        } catch (error) {
+            // do nth
```

Comment:
lines 204-205 can be placed in the catch block, but the result is the same in this situation either way